packages/ui/react-ui-table/src/model/table-controls.ts

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { type MouseEvent } from 'react';
+
+import { type TableModel } from './';
```

Comment:
bad imports:
- no import '.' or 'foo/bar'

since this happens quite frequently (i assume via zed?) can you consider creating a lint rule for this.